fractions/swagger/pom.xml

```diff
@@ -95,6 +95,19 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
```

Comment:
@kenfinnigan I am not sure if this is the right way to bring in the WF modules for module resolution or not, but I saw this in other places and used the same. Please advise if this is incorrect.